.gitignore

```diff
@@ -1,2 +1,4 @@
 vendor
 composer.lock
+
+.idea/
```

Comment:
please remove

Reply:
PhpStorm is probably the most common PHP IDE currently. One of the main goals of the gitignore is to prevent commiting data, that does not belong to the project. I removed it now, but the effect is, that you have to add every single file/folder by hand to not accidentally commit the IDE metadata:

![selection_046](https://user-images.githubusercontent.com/2984003/41770014-4bd5838a-7611-11e8-84e3-09b2307a1071.png)